src/scripts/store/dimStoreItem.directive.template.html

```diff
@@ -17,8 +17,8 @@
        class="item-stat item-quality"
        ng-style="::vm.item.quality.min | qualityColor">{{ ::vm.item.quality.min }}%</div>
   <div ng-if="vm.item.dtrRating"
-       class="item-dtr-rating"
-       ng-style="vm.item.dtrRating | dtrRatingColor">{{ vm.item.dtrRating }}</div>
+       class="item-stat item-review"
+       ng-style="vm.item.dtrRating | dtrRatingColor">{{ ::vm.item.dtrRating }}<i class='fa fa-star'></i></div>
```

Comment:
It shouldn't really ever change, I don't think. The user's review doesn't modify it for another up to 10 mins